CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
The 'Salary' property should be nullable if it's optional, or marked with [Required] if mandatory. This ensures clarity in validation and database schema generation.

```suggestion
public decimal? Salary { get; set; }
```